### PR TITLE
Depart on a friday

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review
#303
* changes the departure date so that it is a Friday

### CODE REVIEW

**Comment by nelsonic** [comment]
@jackcarlisle this _looks_ like it should do the trick.
would you mind adding a test for the new method?

**Comment by jackcarlisle** [comment]
@nelsonic sorry of course!

File: src/reducers/travel-info.js [hunk 1]
```diff
@@ -19,14 +20,15 @@ export const initialState = {
...
-  departureDate: moment().add(3, 'months').format('YYYY-MM-DD'),
+  departureDate: departOnFriday(moment().add(3, 'months')).format('YYYY-MM-DD'),
   passengerBirthdays: [],
   numberOfChildrenTitle: '0',
   numberOfAdultsTitle: '2',
   durationTitle: '1 uger'
 };
 
 export default function travelInfo (state = initialState, action) {
+  console.log('****************', moment().add(3, 'months'));
```
Comment (nelsonic) [inline src/reducers/travel-info.js]: we should probably remove this `console.log` ...

**Comment by nelsonic** [comment]
thanks @jackcarlisle 👍

**Pull request merged**